doc/ko_run.md

```diff
@@ -44,6 +44,7 @@ ko run IMPORTPATH [flags]
   -P, --preserve-import-paths    Whether to preserve the full import path after KO_DOCKER_REPO.
       --push                     Push images to KO_DOCKER_REPO (default true)
       --sbom string              The SBOM media type to use (none will disable SBOM synthesis and upload, also supports: spdx, cyclonedx, go.version-m). (default "spdx")
+      --sbom-dir string          The directory to write the SBOM to.
```

Comment:
```suggestion
      --sbom-dir string          Path to file where the SBOM will be written.
```

I'd like to think about making this more consistent with `--image-refs` if possible, which takes a path to a file, instead of a directory.

That may bite us later if/when we end up generating multiple SBOMs, but I'm not sure we'll do that anyway. If we do, we can only write the first one to disk or something. WDYT?

Reply:
>I'd like to think about making this more consistent with --image-refs if possible, which takes a path to a file instead of a directory.
That's also ok for me. It should change the logic of determining the file path, that's all (I guess?). But what if I'm building more than one binary configured via the [builds](https://github.com/ko-build/ko#overriding-go-build-settings] section?

Reply:
Is there anything left that you want me to do? Or are we all good 🕺🏻

Reply:
What do you think about adding an e2e test in e2e.yaml? You could even have it `--push=false` with `--sbom-dir` then check that the file exists and can be read with `jq . <file>`.

Reply:
I have added a test, can you please take a look at them? Thx